### PR TITLE
DDCNLS-3908: content updated and error updated

### PR DESCRIPTION
**Pull request by Muhammad-Ali-Ahmad**

### CODE REVIEW

**Comment by danewatts** [comment]
@Muhammad-Ali-Ahmad this PR has been sat about for a bit. Ticket looks to be sat in Ready for test. We are trying to handover GMP - can you/we discuss wether this is still needed

File: conf/messages [hunk 1]
```diff
@@ -148,9 +148,9 @@ gmp.year = Year
 gmp.inflation_proofed = Enter the date.
 gmp.caption = Recent calculations
 gmp.revaluation.rate = Revaluation rate
-gmp.revaluation_rate.rate_held_by_hmrc = The rate held by HMRC
+gmp.revaluation_rate.rate_held_by_hmrc = HMRC's rate
```
Comment (oc-ste) [inline conf/messages]: Are we not scrapping contractions? Should this be `HMRC is`

File: conf/messages [hunk 2]
```diff
@@ -59,8 +59,8 @@ gmp.survivor.dol.question = Had the member already left the scheme at date of de
 gmp.fileupload.header = Upload your file
 gmp.bulk.results.banner = Calculation ready
 gmp.bulk_request_received.banner = Your request has been received
-gmp.revaluation_rate.header = Which revaluation rate would you like to use?
-gmp.revaluation_rate_spa.header = If HMRC needs to revalue this calculation which rate would you like to use?
+gmp.revaluation_rate.header = What revaluation rate do you want HMRC to use?
+gmp.revaluation_rate_spa.header = What revaluation rate do you want HMRC to use?
```
Comment (oc-ste) [inline conf/messages]: These are the same, you can delete one

File: test/controllers/RevaluationRateControllerSpec.scala [hunk 3]
```diff
@@ -195,7 +195,7 @@ class RevaluationRateControllerSpec extends PlaySpec with OneServerPerSuite with
               RevaluationRate(None)
             )
             val result = TestRevaluationRateController.post(FakeRequest().withJsonBody(postData))
-            contentAsString(result) must include(Messages("gmp.error.reason.mandatory"))
```
Comment (oc-ste) [inline test/controllers/RevaluationRateControllerSpec.scala]: Can you now delete the message key: gmp.error.reason.mandatory ?

File: conf/messages (hunk 1 above)
Comment (Muhammad-Ali-Ahmad) [reply conf/messages]: "In this context this isn't technically a contraction" Says content designer

File: test/controllers/RevaluationRateControllerSpec.scala (hunk 3 above)
Comment (Muhammad-Ali-Ahmad) [reply test/controllers/RevaluationRateControllerSpec.scala]: it is used in InflationProofForm and DateOfLeavingForm

File: conf/messages (hunk 2 above)
Comment (Muhammad-Ali-Ahmad) [reply conf/messages]: done

**Pull request merged**